samples/lj-demo.py

```diff
@@ -132,7 +136,7 @@
 system.analysis.dist_to(0)
 
 act_min_dist = system.analysis.min_dist()
-system.cell_system.max_num_cells = 2744
+system.cell_system.max_num_cells = 14**3
```

Comment:
where does this magic number come from?

Reply:
I've removed all occurrences of `max_num_cells`. There was no performance impact.